pandas/io/gbq.py

```diff
@@ -215,7 +215,7 @@ def to_gbq(
     table_schema: list[dict[str, str]] | None = None,
     location: str | None = None,
     progress_bar: bool = True,
-    credentials=None,
+    credentials: Any = None,
```

Comment:
The documentation says, it needs to be `google.auth.credentials.Credentials | None`

Reply:
(but if the CI doesn't install the package, there is not much benefit of importing it in a TYPE_CHECKING block.)